docs/common/migrations.md

```diff
@@ -29,9 +29,11 @@ These SQL statements are run by the `Database.Schema.migrate()` method. Migratio
 
 ## Verifying Migrations
 
-You can also place a `.db` file in the `src/main/sqldelight` folder of the same `<version number>.db` format. If there is a `.db` file present, a new `verifySqlDelightMigration` task will be added to the gradle project, and it will run as part of the `check` task, meaning your migrations will be verified against that `.db` file. It confirms that the migrations yield a database with the latest schema.
+A `verifySqlDelightMigration` task will be added to the gradle project, and it will run as part of the `check` task. For any `.db` file named `<version number>.db` in your SqlDelight source set (e.g. `src/main/sqldelight`) it will apply all migrations starting from `<version number>.sqm`. and confirms that the migrations yield a database with the latest schema.
```

Comment:
```suggestion
A `verifySqlDelightMigration` task will be added to the gradle project, and it will run as part of the `check` task. For any `.db` file named `<version number>.db` in your SqlDelight source set (e.g. `src/main/sqldelight`) it will apply all migrations starting from `<version number>.sqm`, and confirms that the migrations yield a database with the latest schema.
```